app/src/sharedTest/java/org/oppia/app/topic/review/ReviewActivityTest.kt

```diff
@@ -0,0 +1,65 @@
+package org.oppia.app.topic.review
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import kotlinx.coroutines.CoroutineDispatcher
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.util.threading.BackgroundDispatcher
+import org.oppia.util.threading.BlockingDispatcher
+import javax.inject.Singleton
+
+/** Tests for [ReviewActivity]. */
+@RunWith(AndroidJUnit4::class)
+class ReviewActivityTest {
+
+  @Test
+  fun testReviewActivity_loadReviewFragment_hasDummyString() {
+    ActivityScenario.launch(ReviewActivity::class.java).use {
+      onView(withId(R.id.dummy_text_view)).check(matches(withText("This is dummy TextView for testing")))
```

Comment:
Please update this with tests corresponding to showing the concept card.

Reply:
Removed this TestActivity because it was covered in `ConceptCardFragmentTest`